### ISSUE TITLE
How to specify region in virtual device?

### ISSUE THREAD

**Issue by MedadRufus** [helium/virtual-lorawan-device#27]
With the option now to specify region(4117ac0016c9ece70f5a7c4a16095c9ea90153f4), how to specify the region in `settings.toml`? Will it be an entry like so in the example below? I think it will be good to add to the `readme.md`.

Listing of an example `settings.toml`
```
region = "EU868"  # specify region, either EU868 or US915

[packet_forwarder.pf_one]
mac = "0807060504030201"
host = "127.0.0.1:1691"

[packet_forwarder.pf_two]
mac = "0807060504030202"
host = "127.0.0.1:1692"
...

```

**Comment by lthiery** [comment]
Hi, @MedadRufus! Thanks for bringing this up. I've added [a section to the README}(https://github.com/helium/virtual-lorawan-device#region) as an explainer

**Comment by MedadRufus** [comment]
Many thanks.
Fixed in 431913a4f273b215eb4a0e211f25b6d32ca61026
